homework-submission/react-hyf/react-hyf-client/src/folders/week3/HomeWorkWeek3.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import ProfilPicture from '../../components/week3/ProfilPicture';
+import UserInfo from '../../components/week3/UserInfo';
+import './style.css';
+
+function HomeWorkWeek3() {
+  const [user, setUser] = useState([]);
+  const [index, setIndex] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+  const randomNum = Math.floor(Math.random() * 10);
+  const userValues = Object.values(user);
+  useEffect(() => {
+    const getUsers = async () => {
+      const res = await fetch(`https://uinames.com/api/?amount=10&&ext=photos`);

```

Comment:
Would be good to show somewhere that new data is being loaded, now after pressing space it takes a bit of time and then the data suddenly changes.

Reply:
It was shown only small div at the top of the window. Now it is bigger and used spinner